rest-services/it.tests/src/test/java/com/_4point/aem/docservices/rest_services/it_tests/server/forms/ExportDataTest.java

```diff
@@ -109,13 +138,12 @@ void testExportData_Bytes() throws IOException {
 			assertTrue(result.hasEntity(), "Expected the response to have an entity.");
 			assertEquals(Response.Status.OK.getStatusCode(), result.getStatus(), ()->"Expected response to be 'OK', entity='" + TestUtils.readEntityToString(result) + "'.");
 			byte[] resultBytes = IOUtils.toByteArray((InputStream)result.getEntity());
-			assertThat("Expected a xml to be returned.", ByteArrayString.toString(resultBytes, 8), containsString("%, X, M, L,-, 1, ., 7"));
+		
 			if (SAVE_RESULTS) {
-				IOUtils.write(resultBytes, Files.newOutputStream(TestUtils.ACTUAL_RESULTS_DIR.resolve("ExportedData.xml")));
+				IOUtils.write(resultBytes, Files.newOutputStream(TestUtils.ACTUAL_RESULTS_DIR.resolve("ExportedData3.xml")));
 			}
 			assertEquals(MediaType.APPLICATION_XML_TYPE, MediaType.valueOf(result.getHeaderString(HttpHeaders.CONTENT_TYPE)));
 		}
 	}
 
-
-}
+	}
```

Comment:
This looks to me like this is actually multiple tests within a single method.  It's a best practice to separate out individual tests into their own methods.  There are multiple reasons for this:

1. If one test fails, the other tests still run.  This way the person running the test can understand the scope of the issue from the beginning, rather than having a test fail, fixing the problem and then finding out the next test is also failing.
1. The tests can be run in parallel.  When test suites grow large, a common tactic to reduce the length of time it takes to run them is to parallelize the individual tests.   JUnit can do this but it can only do this if the tests are each in individual methods.
1. Someone can run a single test without invoking the others.  If only one of the test is failing, and someone is working to correct this, time can be saved by only running the one test that was failing during the testing/debugging/fixing cycle.